src/special-operators.jl

```diff
@@ -289,3 +289,5 @@ function BlockDiagonalOperator(ops...; S = promote_type(storage_type.(ops)...))
   args5 = all((has_args5(op) for op ∈ ops))
   CompositeLinearOperator(T, nrow, ncol, symm, herm, prod!, tprod!, ctprod!, args5, S = S)
 end
+
+# TODO: overload the above for matrices?
```

Comment:
Can you remove this comment and open the issue for some functionalities that are not supported with matrices?

Reply:
I will merge the PR after that 🙂

Reply:
Comment removed, issue opened in # TODO: overload the above for matrices?